hbase-server/src/main/java/org/apache/hadoop/hbase/master/snapshot/SnapshotManager.java

```diff
@@ -654,14 +654,16 @@ private void takeSnapshotInternal(SnapshotDescription snapshot) throws IOExcepti
     TableName snapshotTable = TableName.valueOf(snapshot.getTable());
     if (master.getTableStateManager().isTableState(snapshotTable,
         TableState.State.ENABLED)) {
-      LOG.debug("Table enabled, starting distributed snapshot.");
+      LOG.debug("Table enabled, starting distributed snapshots for {}",
```

Comment:
nit: This still requires the isDebugEnabled() guard because ClientSnapshotDescriptionUtils.toString(..) is not a trivial call.